### PR TITLE
KEYCLOAK-16360 Add realm theme settings to realm CRD

### PR DESCRIPTION
**Pull request by dhx-teuto**
## JIRA ID
https://issues.redhat.com/browse/KEYCLOAK-16360

This pull request is only intended for the last commit, the others are minor fixes to code generation etc. that should be done beforehand.
I will rebase as needed.

## Checklist:
- [ ] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

**Comment by slaskawi** [comment]
I'll hold off with comments until we get your previous PRs in. Then, I'll ask you for a rebase and we'll take it from there.

**Comment by dhx-teuto** [comment]
@slaskawi rebased.

**Comment by slaskawi** [comment]
I've re-triggered the CI to pick the latest changes up.

@miquelsi May I kindly ask you to review this one?

**Review by miquelsi — APPROVED** [review]

File: test/e2e/keycloak_realm_test.go [hunk 1]
```diff
@@ -79,6 +79,13 @@ func getKeycloakRealmCR(namespace string) *keycloakv1alpha1.KeycloakRealm {
 					"envelopeFrom":    "sso@example.com",
 					"ssl":             "",
 				},
+				InternationalizationEnabled: &[]bool{true}[0],
+				SupportedLocales:            []string{"en", "de"},
+				DefaultLocale:               "en",
+				LoginTheme:                  "keycloak",
+				AccountTheme:                "keycloak",
+				AdminTheme:                  "keycloak",
+				EmailTheme:                  "keycloak",
```
Comment (dhx-teuto) [inline test/e2e/keycloak_realm_test.go:88]: @slaskawi

**Comment by dhx-teuto** [comment]
@slaskawi wanna merge this one as well? freshly rebased and I added example values in the test like on the other two.

**Review by slaskawi — APPROVED** [review]

**Comment by slaskawi** [comment]
Integrated, thanks @dhx-teuto !

**Comment by buehlmann** [comment]
Hi @slaskawi. What is the easiest way to find out if this fix is part of the current version (7.4-28) of the RH-SSO operator? I tried to figure it out by the JIRA labels - but without success.

**Pull request merged**